### ISSUE TITLE
Want to move to the graphblas org on docker hub?

### ISSUE THREAD

**Issue by michelp** [bergkvist/docker-graphblas#1]
Hi @bergkvist.  There is a `graphblas` organization on docker hub, it currently only hosts pygraphblas, but it's there for anyone in the community to push their images.  If you'd like to push yours to the org instead of your personal hub account, I can make you an org maintainer.

https://hub.docker.com/orgs/graphblas/repositories

There are currently some old lagraph and suitesparse images in there I would propose deprecating in favor of your image.

**Comment by bergkvist** [comment]
Sure! Feel free to make me an org maintainer - and I'll push this image to the graphblas organization.

This image isn't meant to be used standalone though - and have to be copied into another image to be used.

**Comment by michelp** [comment]
Done! you should see graphblas now in the organization drop down on docker hub.

**Comment by michelp** [comment]
About your standalone comment that's fine, I actually would like to use your image as a base copy image for the pygraphblas minimal and notebook images.

**Comment by bergkvist** [comment]
Thanks!

As a sidenote, I've noticed that graphblas recently has been added to conda-forge for linux and osx.
https://github.com/conda-forge/graphblas-feedstock

**Comment by bergkvist** [comment]
And what do you think about naming convention?

Should I split this into two docker images overwriting these two images? (Or just keep the name lib-lagraph?)
![image](https://user-images.githubusercontent.com/410028/90157743-681c4880-dd8e-11ea-8a9f-29e0ebf0ab25.png)

**Comment by michelp** [comment]
Those images were an experiment of mine to duplicate what you've done better here, they were never announce or to my knowledge used by anyone but me so I'm fine with deleting them.

As for splitting the image, my initial thinking was that LAGraph is still in a state of rapid changing with no fixed release in the future but SuiteSparse is pretty stable up to and beyond 4.0, so maybe it does make sense to split them given their different volatility.

**Comment by bergkvist** [comment]
Yeah, I ended up splitting them. I have now built/released the following on docker hub:
- [graphblas/graphblas](https://hub.docker.com/r/graphblas/graphblas) (v3.3.3 and v4.0.0draft4)
    - latest == v3.3.3
- [graphblas/lagraph](https://hub.docker.com/r/graphblas/lagraph) (13June2020 and 8Aug2020)
    - latest == 13June2020 (overwrote the previous latest-version you published)

Since v4.0.0draft4 of GraphBLAS is a pre-release, and 8Aug2020 of LAGraph depends on the pre-release, I've set latest to be v3.3.3/13June2020 for now.

Building graphblas takes quite a long time, while building lagraph is fast. And since LAGraph might get more frequent updates as you said, this makes sense. I added you as a collaborator to this repo in case you want to make some changes.

**Comment by michelp** [comment]
Sounds good, I deleted the last old image.  Do you want to announce it on the graphblas mailing list?

**Comment by bergkvist** [comment]
Yeah, sure! How do I do that, and how do I subscribe to this mailing list myself?

**Comment by michelp** [comment]
It's not very well advertised, If you email Aydın Buluç abuluc@lbl.gov he can add you to the list.
